src/extension.ts

```diff
@@ -61,12 +61,13 @@ function loadConfiguration() {
 function startLanguageServer(context: vscode.ExtensionContext) {
     // let debugOptions = { execArgv: ["--nolazy", "--debug=6004"] };
 
-    var originalJuliaPkgDir = process.env.JULIA_PKGDIR ? process.env.JULIA_PKGDIR : path.join(os.homedir(), '.julia', 'v0.5');
+    var originalJuliaPkgDir = process.env.JULIA_PKGDIR ? process.env.JULIA_PKGDIR : path.join(os.homedir(), '.julia', 'v0.6');
```

Comment:
This should stay 0.5 for now, I guess. I think once we have merged this PR we can do better here to detect the correct folder, I think I know how that should be done. But for now, lets just keep this at 0.5 and then merge this, ok?